common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayLocation.scala

```diff
@@ -44,6 +44,11 @@ case class DisplayDigitalLocation(
     `type` = "String",
     description = "Who to credit the image to"
   ) credit: Option[String] = None,
+  @Schema(
+    `type` = "String",
+    description =
+      "Text that can be used when linking to the item - for example, 'View this journal' rather than the raw URL"
+  ) link_text: Option[String] = None,
```

Comment:
Any reason we're using `link_text` over `linkText`? I saw it in the RFC, should have asked there.

Reply:
The reason is that I was doing all the experiments in Python, and forgot we used camelCase in the catalogue. Fixing…